bot/exts/utils/subscription.py

```diff
@@ -0,0 +1,182 @@
+        elif all(role_check):
+            embed = Embed(
+                title=f"{Emojis.confirmation_emoji} Subscribed",
+                description=f"You've subscribed to {ctx.guild}'s announcements and polls.",
+                color=Colours.green,
+            )
+            await ctx.send(content=ctx.author.mention, embed=embed)
+        elif role_check == (True, False):
+            embed = Embed(
+                title=f"{Emojis.confirmation_emoji} Subscribed",
+                description=f"You've subscribed to {ctx.guild}'s announcements.",
+                color=Colours.green,
+            )
+            await ctx.send(content=ctx.author.mention, embed=embed)
+        elif role_check == (False, True):
+            embed = Embed(
+                title=f"{Emojis.confirmation_emoji} Subscribed",
+                description=f"You've subscribed to {ctx.guild}'s polls.",
+                color=Colours.green,
+            )
+            await ctx.send(content=ctx.author.mention, embed=embed)
+
+    async def unsubscribe_group_helper(self, ctx: Context) -> None:
+        """Helper function for unsubscribe_group."""
+        roles = (Roles.announcements, Roles.polls)
+        role_check = (
+            await self.de_apply_role(ctx, roles[0]),
+            await self.de_apply_role(ctx, roles[1]),
+        )
+
+        if not any(role_check):
+            embed = Embed(
+                title=f"{Emojis.warning_emoji} Already unsubscribed",
+                description=f"You're already unsubscribed to {ctx.guild}'s announcements and polls.",
+                color=Colours.green,
+            )
+            await ctx.send(content=ctx.author.mention, embed=embed)
+        elif all(role_check):
+            embed = Embed(
+                title=f"{Emojis.confirmation_emoji} Unsubscribed",
+                description=f"You've unsubscribed to {ctx.guild}'s announcements and polls.",
+                color=Colours.green,
+            )
+            await ctx.send(content=ctx.author.mention, embed=embed)
+        elif role_check == (True, False):
+            embed = Embed(
+                title=f"{Emojis.confirmation_emoji} Unsubscribed",
+                description=f"You've unsubscribed to {ctx.guild}'s announcements.",
+                color=Colours.green,
+            )
+            await ctx.send(content=ctx.author.mention, embed=embed)
+        elif role_check == (False, True):
+            embed = Embed(
+                title=f"{Emojis.confirmation_emoji} Unsubscribed",
+                description=f"You've unsubscribed to {ctx.guild}'s polls.",
+                color=Colours.green,
+            )
+            await ctx.send(content=ctx.author.mention, embed=embed)
+
+    @commands.group(name="subscribe", invoke_without_command=True)
+    async def subscribe_group(self, ctx: Context) -> None:
+        """Subscribe to announcements and polls notifications by assigning yourself the role."""
+        await self.subscribe_group_helper(ctx)
```

Comment:
```suggestion
    async def subscribe_group(self, ctx: Context) -> None:
        """Subscribe to announcements and polls notifications by assigning yourself the role."""
        roles = (the tuple)
        await self.subscribe_group_helper(ctx, roles)
```
(according to this  [proposed change](https://github.com/gurkult/gurkbot/pull/71/files#diff-f3b2625c9076cca23a5c94b8b7d62f99df3830a77e0707be8809fcb7c9636250R77)